lms/envs/common.py

```diff
@@ -3043,15 +3051,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 CERT_NAME_LONG = "Certificate of Achievement"
 
 #################### OpenBadges Settings #######################
+# Full guide for setting up OpenBadges available here:
+# https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_badging.html
+#
 
+# .. setting_name: BADGING_BACKEND
+# .. setting_default: 'lms.djangoapps.badges.backends.badgr.BadgrBackend'
+# .. setting_description: The backend service class (or callable) for creating OpenBadges. It must implement
+#    the interface provided by lms.djangoapps.badges.backends.base.BadgeBackend
 BADGING_BACKEND = 'lms.djangoapps.badges.backends.badgr.BadgrBackend'
```

Comment:
What do think about adding a `setting_warning` that refers back to `FEATURES['ENABLE_OPENBADGES']` for each of these?  
Note: `setting_warning` is singular (for now), and isn't yet consistent with `toggle_warnings`.  See https://github.com/edx/code-annotations/blob/master/code_annotations/contrib/config/setting_annotations.yaml#L9